system_files/kinoite/usr/lib/systemd/system/usr-share-sddm-themes.mount

```diff
@@ -0,0 +1,15 @@
+# Workaround to allow KDE Discover to install sddm themes
+
+[Unit]
+Description=KDE writable sddm workaround
+RequiresMountsFor=/usr/share/sddm/themes
+PartOf=bazzite-kde-themes-workaround.target
+
+[Mount]
+Type=overlay
+What=overlay
+Where=/usr/share/sddm/themes
+Options=lowerdir=/usr/share/sddm/themes,upperdir=/var/sddm_themes/themes,workdir=/var/sddm_themes/themes.work
```

Comment:
This probably mount also should probably automatically create that /var/sddm_themes directory too tho, else it'll be mounting a non-existant folder :p

Reply:
I've added mkdir command to create the the working directory and the normal directory, I also ended up removing the PartOf= section since we don't want to recreate the bazzite target